app/src/test/java/com/claraberriel/mvpparking/mvp/presenter/ReservationPresenterTest.java

```diff
@@ -0,0 +1,288 @@
+package com.claraberriel.mvpparking.mvp.presenter;
+
+import com.claraberriel.mvpparking.entities.Parking;
+import com.claraberriel.mvpparking.entities.Reservation;
+import com.claraberriel.mvpparking.mvp.model.ReservationModel;
+import com.claraberriel.mvpparking.mvp.view.ReservationView;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.robolectric.RobolectricTestRunner;
+
+import java.util.Calendar;
+import java.util.Date;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(RobolectricTestRunner.class)
```

Comment:
There is no need to run this test using Robolectric as ReservationPresenter doesn't use any classes from the Android SDK. Robolectric is useful for some integration tests, but they're much slower than unit tests and avoided if possible.

Reply:
if you do so, you may need to put back the mockStatic **Log**

Reply:
I need help with this